ReactWindows/ReactNative/Views/DatePicker/ReactCalendarDatePickerManager.cs

```diff
@@ -0,0 +1,221 @@
+﻿using System;
+using Windows.UI.Xaml.Controls;
+using ReactNative.UIManager;
+using ReactNative.UIManager.Annotations;
+using ReactNative.UIManager.Events;
+using Newtonsoft.Json.Linq;
+
+namespace ReactNative.Views.DatePicker
+{
+    /// <summary>
+    /// A view manager responsible for rendering a CalendarDatePicker.
+    /// </summary>
+    public class ReactCalendarDatePickerManager : BaseViewManager<CalendarDatePicker, ReactCalendarDatePickerShadowNode>
```

Comment:
Not really, I just used the code for some components as a reference, and they just happened to use BaseViewManager instead. What are the advantages of it? Do you have any documentation I can refer to?